HandsOnTutorial/300-distributed-multicore-parallel-search.fsx

```diff
@@ -17,8 +22,8 @@ open MBrace.Flow
  Before running, edit credentials.fsx to enter your connection strings.
 **)
 
-(** First you connect to the cluster: *)
-let cluster = MBraceAzure.GetHandle(config)
+#load "lib/collections.fsx"
+#load "lib/mersenne.fsx"
```

Comment:
The top declarations are omitted during document generation, so I thought it best to keep implementation-specific loads visible.